src/plan/aggregate.rs

```diff
@@ -153,39 +174,39 @@ impl<P: Implementable> Implementable for Aggregate<P> {
                         Some((key, DiffPair::new(v, 1)))
                     })
                     .count()
-                    .map(|(key, diff_pair)| {
+                    .map(move |(key, diff_pair)| {
                         let mut v = key.clone();
                         v.push(Value::Rational32(Ratio::new(
                             diff_pair.element1 as i32,
                             diff_pair.element2 as i32,
                         )));
-                        v
-                    }),
-            },
-            AggregationFn::VARIANCE => SimpleRelation {
```

Comment:
what happened to Variance?

Reply:
I see. Yeah let's replace Rational32 with Rational64 entirely, this is 2018 :P

Reply:
Should I do another PR for the Rational32 -> Rational64?
Or add it here as a commit

Reply:
Done